pages/api/v1/auth/actions/addPostComment.ts

```
@@ -32,21 +34,38 @@ const handler: NextApiHandler<IAddPostCommentResponse | MessageType> = async (re
 
 	if (!network || !isValidNetwork(network)) return res.status(400).json({ message: 'Missing network name in request headers' });
 
-	const { userId, content, postId, postType, sentiment, trackNumber = null } = req.body;
+	const { userId, content, postId, postType, sentiment, trackNumber = null, isExpertComment, userAddress } = req.body;
 	if (!userId || !content || isNaN(postId) || !postType) return res.status(400).json({ message: 'Missing parameters in request body' });
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)